satwatch/umbrel-app.yml

```
@@ -0,0 +1,31 @@
+manifestVersion: 1.1
+id: satwatch
+category: bitcoin
+name: satwatch
+version: "1.1.5"
+tagline: Bitcoin monitoring and real-time alerts
+description: >-
+  sat.watch provides real-time monitoring of your Bitcoin.
+
+  Features:
+    - Monitor Bitcoin addresses, xpubs, or descriptors
+    - Alerts for funds sent or received
+    - Monitors the mempool and confirmed blocks
+    - Total balance summary for all addresses monitored
+    - Email and webhook notifications
+    - PGP encrypted email
+developer: J.P. Cummins
+releaseNotes: ""
+dependencies:
+  - bitcoin
+  - electrs
+website: https://sat.watch
+gallery: []
+repo: https://github.com/jpcummins/sat.watch
+support: https://github.com/jpcummins/sat.watch/issues
+port: 3883
+path: ""
+defaultUsername: "satwatch"
+defaultPassword: "satwatch"
```

Show deterministic password on app page.
```suggestion
deterministicPassword: true
```